cachito/web/static/api_v1.yaml

```diff
@@ -1058,14 +1139,20 @@ components:
           example: Completed successfully
         packages_count:
           type: integer
+          minimum: 0
           example: 1
         dependencies_count:
           type: integer
+          minimum: 0
           example: 100
         error_origin:
           type: string
+          enum:
+            - client
+            - server
         error_type:
           type: string
+      additionalProperties: false
```

Comment:
Another thing to validate could be `minProperties: 1` (if openapi supports that)

Reply:
Will add